stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: nginx-ingress
 version: 0.8.22
```

Comment:
The chart version needs to be incremented. This is causing a CI failure at the moment.